lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -230,13 +230,50 @@ def publishing_api_does_not_have_item(content_id)
         stub_request(:get, url).to_return(status: 404, body: resource_not_found(content_id, "content item").to_json, headers: {})
       end
 
+      # Stubs a request to links endpoint
+      #
+      # Requires a hash as paramenter
+      #
+      # @param links [Hash]
```

Comment:
I think YARD likes this in the form `@param [Type] var_name And then a description`